pkg/project/auth/cache.go

```diff
@@ -253,9 +186,13 @@ func (ac *AuthorizationCache) synchronizeNamespaces(userSubjectRecordStore cache
 
 // synchronizePolicies synchronizes access over each policy
 func (ac *AuthorizationCache) synchronizePolicies(userSubjectRecordStore cache.Store, groupSubjectRecordStore cache.Store, reviewRecordStore cache.Store) {
-	items := ac.policyIndexer.List()
-	for i := range items {
-		policy := items[i].(*authorizationapi.Policy)
+	policyList, err := ac.policyCache.Policies().List()
+	if err != nil {
+		util.HandleError(fmt.Errorf("Error getting policies: %v", err))
```

Comment:
Ugh, not a fan of this method, but if you're going to do it, pass the error straight through. Without a causedby for proper chaining, error type identification becomes impossible if you're wrapping.